test/ufuzz.js

```diff
@@ -440,7 +502,7 @@ for (var round = 0; round < num_iterations; round++) {
     var beautify_result = run_code(beautify_code);
 
     try {
-      var uglify_code = minify(beautify_code, {
+      var uglify_code = minify(original_code, {
```

Comment:
IIRC there was a prior discussion involving @kzc and we ended up keeping this unchanged.

If we are going to change this, might as well get rid of `beautify_result` and move computation of `beautify_code` into `log()`. That way you should get a nice speed-up.